avl_tree/AVL_Tree.java

```diff
@@ -0,0 +1,294 @@
+import java.util.Scanner;
+
+public class AVL_Tree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		int a;
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
I used break statement there also...loop has to repeat until it cannot reach break statement. There is no sense of make a boolean variable and use it as argument in loop.